README.md

```diff
@@ -1 +1,19 @@
-# provardx-vscode
\ No newline at end of file
+# ProvarDX SFDX Plugin Integration for Visual Studio Code
+
+This extension enables Visual Studiio Code to run ProvarDX CLI commands in an interactive way. Existing Provar customers can execute
+Provar Test Cases from Visual Studio Code. You must be an existing Provar customer with a valid paid license to write and maintain your test cases.
+
+![GIF showing Create properties file, validate and running tests](/images/provardx-preview.gif)
+
+## Prerequisites
+
+Before you set up ProvarDX CLI Integration for VS Code, make sure that you have these essentials.
+
+-   **Provar**  
```

Comment:
Do we need both Provar Desktop and ProvarDX installed to use this?

Reply:
Yes we would need both.